examples/low-level/crypto/Crypto.Symmetric.Poly1305.MAC.fst

```diff
@@ -130,6 +138,7 @@ val alloc: i:id
     (ideal ==> m_contains (ilog st.log) m1 /\ m_sel m1 (ilog st.log) == None)
   ))
 
+#reset-options "--z3timeout 1000"
```

Comment:
This shouldn't require such a large timeout. Should find out what's taking so much and fix it.